package.json

```diff
@@ -50,28 +50,28 @@
     "javascript-state-machine": "^3.1.0",
     "lodash": "^4.17.15",
     "miniee": "0.0.5",
-    "minilog": "2.0.8",
-    "mobx": "^5.15.0",
+    "minilog": "^3.1.0",
```

Comment:
Careful with Minilog update. It could disable logging entirely.

Reply:
ah yeah, I thought I remembered something about that. How did we test it last time?

Reply:
Last time, I just reverted to the old version when I tested in zendesk_radar.

Reply:
putting it back to 2 until we have time to validate it doesn't cause issue upstream 👍